### ISSUE TITLE
Tarima cannot start

### ISSUE THREAD

**Issue by plmarko** [tacoss/tarima#6]
Hello, 
I have problem to run tarima for pulltorefresh.js repository. I'm trying to run it on windows 10 in git bash.

The error message:
```
$ tarima watch -s
tarima v4.1.0 (node v9.11.2 - development)
› Output to: .
› Reading from 1 source
› Serving files from: .
─ http://localhost:3000
"rror: no parsers registered for: "
Error: ENOENT: no such file or directory, open ''
      bundle - src....html

```

**Comment by pateketrueke** [comment]
Hi, thanks for your feedback.

This module has not been tested on Windows yet, so figuring out what's happening is bit difficult to me to achieve... I think `-s` is being taken as directory or something like that.

Aside of this, I believe you're missing a valid source directory, e.g. `tarima watch sources`

This can be provided on the package.json directly:

```json
{
  ...
  "tarima": {
    "from": "sources"
  }
}
```

**Comment by plmarko** [comment]
Hello, 

I have this in my package.json. 
```
 "tarima": {
    "from": "src",
    ...
}
```
As I mentioned on the top I'm trying to run it in [pulltorefresh.js](https://github.com/BoxFactura/pulltorefresh.js)  project  and with [package.json](https://github.com/BoxFactura/pulltorefresh.js/blob/master/package.json)

I've tried to run first the dev command from script but it failed with some 
`$ npm run dev`

then I've tried to run it directly 
`$tarima watch -s`

but it failed (see error message above). 

I don't know how tarima works but maybe the [cross-env](https://www.npmjs.com/package/cross-env)  might help here to make it run on different OS.

I forgot to mention that I use [NVM for Windows](https://github.com/coreybutler/nvm-windows)

Thank you for your time.
